<launch>
   <node pkg="autosail" name="wind_sensor_node" type="wind_sensor_node.py"/>
</launch>